app/TracingStrategyContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext } from 'react';
+import { TracingStrategy } from './tracingStrategy';
+import { ExposureHistoryProvider } from './ExposureHistoryContext';
+
+interface TracingStrategyContextState {
+  name: string;
```

Comment:
right now, name is acting as a bit of a place-holder. Future commits would like put the strategy specific copy and assets in here.